src/client/Contracts.js

```diff
@@ -572,3 +574,113 @@ exports.SendFunds = async function({recipient, ether}) {
 
   return await transaction.wait();
 };
+
+/**
+ * TenantContractId returns the ID of the tenant contract for
+ * the objectId provided from contract metadata or from fabric metadata
+ *
+ * @param objectAddress
+ * @returns {Promise<string|undefined>}
+ */
+exports.TenantContractId = async function(objectAddress) {
```

Comment:
Small things to change about these methods:
- use named parameters (e.g. `({objectAddress})` instead of `(objectAddress)`
- call that parameter `contractAddress`, that's what most other methods use
- add objectId or versionHash as parameters and convert them to address if present. most other methods take these

Reply:
sure made the chages.